Makefile

```diff
@@ -277,6 +279,8 @@ modules: ## Runs go mod to ensure proper vendoring.
 	cd $(TOOLS_DIR); go mod verify
 	cd $(APIS_DIR); go mod tidy
 	cd $(APIS_DIR); go mod verify
+	cd $(TEST_DIR); go mod tidy
+	cd $(TEST_DIR); go mod verify
```

Comment:
once the other comment is taken care, please run `make modules` from the root of the repo, to make sure all modules up-to-date. It might be that some modules still need an update or downgrade in some cases. If there are changes in go.mod/go.sum files, include them in the commit.

Reply:
Thanks, I've pushed changes to address those comments :) 🤞